packages/manager/apps/nutanix/src/app.module.js

```diff
@@ -48,6 +49,21 @@ export default (containerEl, environment) => {
         });
       },
     )
+    .run(
+      /* @ngInject */ ($translate) => {
+        let lang = $translate.use();
+
+        if (['en_GB', 'es_US', 'fr_CA'].includes(lang)) {
+          lang = lang.toLowerCase().replace('_', '-');
+        } else {
+          [lang] = lang.split('_');
+        }
+
+        return import(`script-loader!moment/locale/${lang}.js`).then(() =>
```

Comment:
webpack configuration should apply following plugin `ContextReplacementPlugin`.
see (as an example): 
https://github.com/ovh/manager/blob/361d0dfaba9d3d8bb531f2765a038b866cbe5571/packages/manager/apps/dedicated/webpack.config.js#L108-L111

Reply:
Yes in dedicated, but in standalone `nutanix` module, we have to add this.